lib/util/tests/ert_util_type_vector_test.cpp

```diff
@@ -263,21 +263,21 @@ void test_resize() {
   int i;
   int def = 77;
   int_vector_type * vec = int_vector_alloc(0,def);
-  int_vector_resize( vec , 10 );
-  test_assert_int_equal( int_vector_size( vec ) , 10 );
+  int_vector_resize( vec , 10 , int_vector_get_default(vec) );
```

Comment:
I guess the `xxx_get_default()` function will have to go as well, so I would just use the `def` variable here, instead of invoking the `int_vector_get_default()` function.